### DISCUSSION TITLE
PS4 Cheats Manager v1.2.2

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/PS4CheatsManager#116] [Announcements]
### Added

- Disable and re-enable cheat files
  - Disabled files won't be loaded by GoldHEN
- Option to change the default URLs for cheats, patches, plugins (`Settings`)

### Changed

- Download Patches from https://github.com/illusion0001/PS4-PS5-Game-Patch/

### Misc

- Includes latest official cheat+patch pack (2024-06-14)

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/PS4CheatsManager/releases/tag/v1.2.2'>PS4 Cheats Manager v1.2.2</a>.</em>

**Comment by ViniciusBCarvalho** [comment]
I'm trying to enable cheats and I can't with this latest version. Patches are working as expected...

**Comment by Bangjoo44** [comment]
Anyone can help me? Get one piece pirate warriors 4 cheats, CUSA17196 V01.30

**Comment by RetrogamerX** [comment]
Hope we see a PS5 release?.  Thank you

**Comment by johnchoi112** [comment]
Dear Mr. Bucanero,

Thank you for the file update. However, I noticed that the update and cheat information related to SRW30 are missing. Could we expect the related files to be uploaded soon?

Best regards

**Comment by Adonis427** [comment]
Por qué cuando intenté instalar el gold hen en la ps4 pro no se instalo 🤔

**Comment by JaydenNioh** [comment]
hey anybody knows why I cant start this on my ps5 (installed via debug settings)

**Comment by laibikassem2009-tech** [comment]
Please add far cry 6 (CUSA15778 v01.10)
